app/src/app/home/pages/suite/item.component.ts

```diff
@@ -53,7 +53,9 @@ export class SuiteItemBatchComponent extends PillContainerComponent {
     this.data = {
       link: meta.slug,
       name: meta.slug.split('@')[0],
-      query: null
+      query: {
+        cv: item.comparedAgainst
+      }
```

Comment:
Is this going to append `?cv=` to versions and elements that are being compared against the baseline? We don't want to do that.

Reply:
Thank you!